### PR TITLE
:package: Fix phpdoc `var -> param` when misused

### PR DESCRIPTION
**Pull request by andrew-demb**

### CODE REVIEW

File: src/Psalm/Internal/Codebase/Analyzer.php [hunk 1]
```diff
@@ -885,7 +885,7 @@ public function hasMixedMemberName(string $member_id) : bool
     }
 
     /**
-     * @var array<string, string<bool>> $names
+     * @param array<string, string<bool>> $names
```
Comment (psalm-shepherd) [inline src/Psalm/Internal/Codebase/Analyzer.php]: Cannot create generic object with reserved word in docblock for Psalm\Internal\Codebase\Analyzer::addMixedMemberNames
```
     * @param array<string, string<bool>> $names
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by psalm-shepherd** [comment]
Psalm also found errors in other files

```
src/Psalm/IssueBuffer.php:79:67 - Argument 2 of Psalm\Config::getreportinglevelforissue expects array<array-key, string>, parent type array<array-key, mixed> provided
```

**Comment by psalm-shepherd** [comment]
Psalm found errors in other files

```
src/Psalm/IssueBuffer.php:79:67 - Argument 2 of Psalm\Config::getreportinglevelforissue expects array<array-key, string>, parent type array<array-key, mixed> provided
```

**Comment by muglug** [comment]
Thanks, fixed in 7fc11af

**Pull request closed without merging**